host/lib/usrp/crimson_tng/io_impl.cpp

```diff
@@ -1015,11 +1017,11 @@ tx_streamer::sptr crimson_tng_impl::get_tx_stream(const uhd::stream_args_t &args
     static const size_t hdr_size = 0
         + vrt_send_header_offset_words32*sizeof(uint32_t)
         + vrt::max_if_hdr_words32*sizeof(uint32_t)
-        - sizeof(vrt::if_packet_info_t().tlr) //crimson tng does not use trailer on tx
```

Comment:
The trailer does not need to be subtracted here as max_if_hdr_words32 does not include the size of the trailer.

Removing this line fixes the off by 1 growth observed in the internal FPGA buffer.